### PR TITLE
[feat] Add error alerts for file fetching

### PR DESCRIPTION
**Pull request by gregorylegarec**
This PR :

Needs this branch of Cozy UI : https://github.com/gregorylegarec/cozy-ui/tree/feat/improve-alerts

* Connects App to redux
* Add Alerter component, which show alerts. Based on `Notifier.jsx` in `MyAccounts` app, but the behaviour is a little bit different (Overlay, reloading, button, etc.)
* At this time, the alerter mechanism only show critical errors.
* To trigger CSS transition, I used a `setTimeout` in `componentDidMount` method. I clearly not a big fan and I'd love to replace it by a better solution.

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
 import Topbar from './Topbar'
 
-const App = ({ t, children }) => (
+const reload = () => {
+  window.location.reload()
+}
+
+const App = ({ t, error, children }) => (
   <div class={classNames(styles['fil-wrapper'], styles['coz-sticky'])}>
+    { error &&
```
Comment (goldoraf) [inline src/components/App.jsx]: I'd prefer a one-liner here but it's only a matter of personal preference :)

File: src/reducers/ui.js [hunk 2]
```diff
@@ -2,17 +2,21 @@ import { combineReducers } from 'redux'
...
 import {
   FETCH_FILES,
+  FETCH_FILES_FAILURE,
   RECEIVE_FILES,
   CREATE_FOLDER,
   CREATE_FOLDER_SUCCESS,
   UPLOAD_FILE,
-  UPLOAD_FILE_SUCCESS
+  UPLOAD_FILE_SUCCESS,
+  RELOAD_PAGE
```
Comment (goldoraf) [inline src/reducers/ui.js]: This const is now useless, right?

File: src/reducers/ui.js [hunk 3]
```diff
@@ -2,17 +2,21 @@ import { combineReducers } from 'redux'
...
 } from '../actions'
 
 const isFetching = (state = false, action) => {
   switch (action.type) {
     case FETCH_FILES:
       return action.folderId
+    case FETCH_FILES_FAILURE:
+      return false
```
Comment (goldoraf) [inline src/reducers/ui.js]: This line isn't necessary: you can pile up 'case' statements when the instructions to be executed are the same.

File: src/reducers/ui.js [hunk 4]
```diff
@@ -51,8 +55,23 @@ const updating = (state = [], action) => {
   }
 }
 
+const error = (state = null, action) => {
+  switch (action.type) {
+    case FETCH_FILES_FAILURE:
+      console.debug('FETCH_FILES_FAILURE')
```
Comment (goldoraf) [inline src/reducers/ui.js]: Forgotten console.debug?

**Comment by gregorylegarec** [comment]
@goldoraf Sorry about the evident lint issue. I updated the two lines of code you mentionned, let's try to unify the code as most as possible. :)

**Pull request merged**